src/test/java/bio/terra/service/filedata/google/firestore/FireStoreCleanProject.java

```diff
@@ -0,0 +1,84 @@
+package bio.terra.service.filedata.google.firestore;
+
+import bio.terra.common.category.Connected;
+import bio.terra.common.fixtures.JsonLoader;
+import bio.terra.service.iam.IamService;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.firestore.QuerySnapshot;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
+@Category(Connected.class)
+public class FireStoreCleanProject {
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private IamService samService;
+
+    @MockBean
+    private TestConfiguration testConfiguration;
+
+    @MockBean
+    private JsonLoader jsonLoader;
+
+    /*
+     * NOTE: This is not a test. This is a tool to delete all FireStore collections from a project.
+     * It is written as a JUnit test because that is the simplest way for me to package it today.
```

Comment:
can you elaborate on why this is simplest? Is there a TODO task to update it?

Reply:
It is the simplest for me to do at that moment. It is definitely not the best. See the poll I just made in the jade-internal slack channel :)